### PR TITLE
capture user facing name and description for all experiment fixes 1124

### PR DESCRIPTION
**Pull request by tiftran**
changes for #1124

### CODE REVIEW

File: app/experimenter/templates/experiments/section_overview.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
   {% if experiment.engineering_owner %}
     <p><strong>Engineering Owner:</strong> {{ experiment.engineering_owner }}</p>
   {% endif %}
+  <p><strong>Public Name:</strong> {{ experiment.public_name }}</p>
+  <p><strong>Public Description:</strong> {{ experiment.public_description }}</p>
```
Comment (jaredkerim) [inline app/experimenter/templates/experiments/section_overview.html]: Ah you have to add a [linebreaks tag](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#linebreaks) here

**Comment by jaredkerim** [comment]
Oh and also 

1) When you submit your PR can you format the commit title as `Capture user facing name and description for all experiment *fixes* 112` ie use the word `fixes` before the bug number that way it automatically closes the issue when you merge

2) You'll have to rebase this to fix conflicts and re generate the migration (I hope I can find a better workflow for this migration numbering conflict situation)

**Comment by tiftran** [comment]
Screenshots below
Screen Shot 2019-04-03 at 10 15 20 AM
Screen Shot 2019-04-03 at 10 15 08 AM

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+## Project Specific Etiquette
 
-## Contributions
-
-For all other questions about contribution see our
-[contributing.md](https://github.com/mozilla/experimenter/blob/master/contributing.md)
-document.
+In some cases, there will be additional project etiquette i.e.: (https://bugzilla.mozilla.org/page.cgi?id=etiquette.html).
+Please update for your project.
+-->
```
Comment (jaredkerim) [inline CODE_OF_CONDUCT.md]: Ah crap you got trapped by the CODE_OF_CONDUCT mishap of 2019.  I think if you rebase this on master this should go away.

File: app/experimenter/experiments/forms.py [hunk 3]
```diff
@@ -384,6 +392,9 @@ class Meta:
             "population_percent",
             "firefox_version",
             "firefox_channel",
+            "client_matching",
```
Comment (jaredkerim) [inline app/experimenter/experiments/forms.py]: `"client_matching"` appears twice here

File: app/experimenter/experiments/models.py [hunk 4]
```diff
@@ -96,6 +96,12 @@ class Experiment(ExperimentConstants, models.Model):
         blank=True,
         null=True,
     )
+
+    public_name = models.CharField(max_length=255, blank=True, null=True)
+
+    public_description = models.CharField(
+        max_length=1024, blank=True, null=True
```
Comment (jaredkerim) [inline app/experimenter/experiments/models.py]: Actually we should probably use TextField here with no max_length

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (tiftran) [reply CODE_OF_CONDUCT.md]: Aww man, something funky went with the initial rebase...
Ended up duplicating the branch, hard resetting the original, pulled and cherry picked 🙈

Comment (jaredkerim) [reply CODE_OF_CONDUCT.md]: The dark side of the Git is a pathway to many abilities some consider to be unnatural.

**Pull request merged**